crates/hyperdrive-math/src/long/close.rs

```diff
@@ -115,4 +159,52 @@ mod tests {
         assert!(result.is_err());
         Ok(())
     }
+
+    // Tests market valuation against yield space valuation when closing a long
+    // with the minimum transaction amount.
+    #[tokio::test]
+    async fn test_calculate_value_long() -> Result<()> {
+        let tolerance = int256!(1e15);
+
+        // Fuzz the spot valuation and yield space valuation against each other.
+        let mut rng = thread_rng();
+        for _ in 0..*FAST_FUZZ_RUNS {
+            let state = rng.gen::<State>();
```

Comment:
We might want to add a big liquidity amount to this to allow us to reduce the tolerance, since we expect it to be more accurate with higher liquidity. Thankfully, we have a function to do this already!

```rs
            let state = rng.gen::<State>().calculate_pool_state_after_add_liquidity(fixed!(1e27), true);
```

Reply:
Ohhh nice! Tried this out, decreased tolerance to 1e13. The error is taken as a percentage, so that's a 0.001% difference between spot and yield space valuations.